src/defaultsettings.cpp

```diff
@@ -565,7 +566,8 @@ void set_default_settings()
 	settings->setDefault("active_block_range", "2");
 	settings->setDefault("viewing_range", "50");
 	settings->setDefault("leaves_style", "simple");
-	settings->setDefault("enable_post_processing", "false");
+	settings->setDefault("enable_post_processing", "false"); // remove this?
```

Comment:
Removing "enable_post_processing = false" as an option would mean dropping support for OpenGL ES 2.0 without extensions afaict (https://github.com/minetest/minetest/issues/13519#issuecomment-2462938547)

(Also: the last time I tested, "enable_post_processing = false" still resulted in a performance improvement on my laptop)

Reply:
That's not the plan. I was only considering enabling it by default.

> the last time I tested, "enable_post_processing = false" still resulted in a performance improvement on my laptop

I hoping that `post_processing_texture_bits` could mend that to some degree